src/lib/serviceRoutines/getAllContextEntities.cpp

```diff
@@ -38,6 +39,22 @@
 /* ****************************************************************************
 *
 * getAllContextEntities - 
+*
+* GET /v1/contextEntities
+*
+* Payload In:  None
+* Payload Out: QueryContextResponse
+*
+* URI parameters:
+*   - attributesFormat=object
+*   - entity::type=XXX
+*   - !exist=entity::type
+*   - exist=entity::type
```

Comment:
Actually I was thinking about that but too lazy to do it ...
But sure, let's add it as a part of the functest